doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -719,60 +338,28 @@ SUIT samples
 Trusted Firmware-M (TF-M) samples
 ---------------------------------
 
-* Replaced support for the ``nrf54l15pdk/nrf54l15/cpuapp/ns`` board target with ``nrf54l15dk/nrf54l15/cpuapp/ns``.
-
-* :ref:`tfm_psa_template` sample:
-
-  * Added support for updating the network core on the nRF5340 DK.
+|no_changes_yet_note|
 
 
 Thread samples
 --------------
 
-* Updated all samples to enable the :ref:`ug_thread_build_report` generation.
-
-* :ref:`ot_cli_sample` sample:
-
-  * Added support for the :ref:`zephyr:nrf54l15dk_nrf54l15` in the low-power snippet.
-  * Added experimental support for :ref:`Trusted Firmware-M <ug_tfm>` on the nRF54L15 SoC.
+|no_changes_yet_note|
 
 Zigbee samples
 --------------
 
-* :ref:`zigbee_light_switch_sample` sample:
-
-  * Added the option to configure transmission power.
-  * Fixed the FOTA configuration for the nRF5340 DK.
+|no_changes_yet_note|
 
 Wi-Fi samples
 -------------
 
-* :ref:`wifi_radio_test` sample:
-
-  * Added capture timeout as a parameter for packet capture.
-  * Expanded the scope of ``wifi_radio_test show_config`` subcommand and rectified the behavior of ``wifi_radio_test tx_pkt_preamble`` subcommand.
-
-* :ref:`softap_wifi_provision_sample` sample:
-
-  * Increased the value of the :kconfig:option:`CONFIG_SOFTAP_WIFI_PROVISION_THREAD_STACK_SIZE` Kconfig option to 8192 bytes to avoid stack overflow.
-
-* :ref:`wifi_shell_sample` sample:
-
-  * Added support for running the full stack on the Thingy:91 X.
-    This is a special configuration that uses the nRF5340 as the host chip instead of the nRF9151.
+|no_changes_yet_note|
```

Comment:
```
* Added overlay to support Wi-Fi Enterprise mode for `shell` sample
```

Reply:
```
+* Added:
+
+  * The :ref:`wifi_offloaded_raw_tx_packet_sample` sample that demonstrates transmission of raw packets.
```

Reply:
Added to file : https://github.com/nrfconnect/sdk-nrf/pull/18192/files#diff-edb5a676e90130a0ac595e21ff13ec34ea32ae0c53dad433bc0d0eab4d10030b.